### DISCUSSION TITLE
I think I deleted most of my .config directory...

### DISCUSSION THREAD

**Discussion by phunni** [twpayne/chezmoi#3375] [Q&A]
I've been having problems getting chezmoi to recognize some of the directories in my .config. I later realised this was because the files I was pointing at were symlinks as a result of a previous solution I'd tried to use. chezmoi simply wouldn't acknowledge that the files were managed or even existed despite "chezmoi managed" listing them.

I thought I could clean it all up by simply removing the files from chezmoi management replacing the symlinks with real files and starting again. So I ran "chezmoi remove .config" because nothing else under .config was managed and now most of my .config directory is gone...

I'm pretty sure I have the important stuff backed up... I really hope I do...

**Comment by Lockszmith-GH** [comment]
🤕 ouch.
I'm guessing you what you wanted to do was `forget` and not `remove`.

A note about this has gone up in the docs [last week](https://github.com/twpayne/chezmoi/commit/47609a3d74b7db45db10805b2e43fa4b86737de3) just about a situation like yours:

https://www.chezmoi.io/reference/commands/forget/

I hope you get things setup up.
